### PR TITLE
Add PTO & h-bridge to Amiga PDO1

### PR DESCRIPTION
**Pull request by Hackerman342**

### CODE REVIEW

**Comment by Hackerman342** [comment]
@edgarriba this is ready to go, but I think it should be kept out of the farm-ng-amiga release until we either:
(1) are ready to release firmware v0.1.9 (necessary to use this feature)
(2) have a need to add it (like someone develops an app or service that controls the actuators)

**Comment by Hackerman342** [comment]
To install farm-ng-amiga from this branch, run:
```bash
pip install farm-ng-amiga@git+https://github.com/farm-ng/farm-ng-amiga.git@pto-in-amigaPDO
```
> Note: there may be a conflict with minimum version requirements in setup.cfg

File: py/farm_ng/canbus/packet.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 from farm_ng.core.stamp import timestamp_from_monotonic
 from farm_ng.core.timestamp_pb2 import Timestamp
 
+# import warnings
```
Comment (edgarriba) [inline py/farm_ng/canbus/packet.py:25]: Remove

**Review by edgarriba — APPROVED** [review]

File: py/farm_ng/canbus/packet.py (hunk 1 above)
Comment (Hackerman342) [reply py/farm_ng/canbus/packet.py:25]: Is coming back soon, but sure

**Pull request merged**